src/main/java/com/cloudant/fdblucene/benchmark/BasicSearchBenchmark.java

```diff
@@ -0,0 +1,70 @@
+package com.cloudant.fdblucene.benchmark;
+
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.lucene.index.Term;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TermQuery;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Group;
+import org.openjdk.jmh.annotations.GroupThreads;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Timeout;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+public class BasicSearchBenchmark {
+
+    @BenchmarkMode(Mode.Throughput)
+    @Fork(1)
+    @Warmup(iterations = 3, time = 10, timeUnit = TimeUnit.SECONDS)
+    @Measurement(iterations = 3, time = 10, timeUnit = TimeUnit.MINUTES)
+    @Timeout(time = 5, timeUnit = TimeUnit.MINUTES)
+    @OutputTimeUnit(TimeUnit.SECONDS)
+    @Benchmark
+    @Group("searchFDB")
+    @GroupThreads(1)
+    public void searchFDB(FDBSearchSetup setup) throws Exception {
+        int randomSearchPosition = setup.random.nextInt(setup.searchTermList.size());
+        String term = setup.searchTermList.get(randomSearchPosition);
+        // we don't actually care about the number of hits
+        setup.searcher.search(new TermQuery(new Term("body", term)), setup.topNDocs);
+    }
+
+    @BenchmarkMode(Mode.Throughput)
+    @Fork(1)
+    @Warmup(iterations = 3, time = 10, timeUnit = TimeUnit.SECONDS)
+    @Measurement(iterations = 3, time = 10, timeUnit = TimeUnit.MINUTES)
+    @Timeout(time = 5, timeUnit = TimeUnit.MINUTES)
+    @OutputTimeUnit(TimeUnit.SECONDS)
+    @Benchmark
+    @Group("searchNIOS")
+    @GroupThreads(1)
```

Comment:
why use Group or GroupThreads when we only have one thread for the tests and only one task in the benchmark?